_partials/_migrate_post_schema_caggs_etal.md

```diff
@@ -0,0 +1,165 @@
+    -f dump_post_data.bak <DATABASE_NAME>
+    ```
+
+1.  Restore the dumped schema post-data from the `dump_post_data.bak` file into
+    your Timescale database, using your connection details. To avoid permissions
+    errors, include the `--no-owner` flag:
+
+    ```bash
+    pg_restore -U tsdbadmin -W \
+    -h <HOST> -p <PORT> --no-owner -Fc \
+    -v -d tsdb dump_post_data.bak
+    ```
+
+</Procedure>
+
+### Troubleshooting
+
+If you see these errors during the migration process, you can safely ignore
+them. The migration still occurs successfully.
+
+```bash
+pg_restore: error: could not execute query: ERROR:  relation "<relation_name>" already exists
+```
+
+```bash
+pg_restore: error: could not execute query: ERROR:  trigger "ts_insert_blocker" for relation "<relation_name>" already exists
+```
+
+## Recreate continuous aggregates
+
+By default, continuous aggregates aren't migrated when you transfer your schema
+and data separately. Restore them by recreating the continuous aggregate
+definitions and recomputing the results on your Timescale database. The recomputed
+continuous aggregates only aggregate existing data in your Timescale database. They
+don't include deleted raw data.
+
+<Procedure>
+
+### Recreating continuous aggregates
+
+1.  Connect to your source database:
+
+    ```bash
+    psql "postgres://<SOURCE_DB_USERNAME>:<SOURCE_DB_PASSWORD>@<SOURCE_DB_HOST>:<SOURCE_DB_PORT>/<SOURCE_DB_NAME>?sslmode=require"
+    ```
+
+1.  Get a list of your existing continuous aggregate definitions:
+
+    ```sql
+    SELECT view_name, view_definition FROM timescaledb_information.continuous_aggregates;
+    ```
+
+    This query returns the names and definitions for all your continuous
+    aggregates. For example:
+
+    ```sql
+    view_name       |                                            view_definition
+    ----------------+--------------------------------------------------------------------------------------------------------
+    avg_fill_levels |  SELECT round(avg(fill_measurements.fill_level), 2) AS avg_fill_level,                                +
+                    |     time_bucket('01:00:00'::interval, fill_measurements."time") AS bucket,                            +
+                    |     fill_measurements.sensor_id                                                                       +
+                    |     FROM fill_measurements                                                                            +
+                    |     GROUP BY (time_bucket('01:00:00'::interval, fill_measurements."time")), fill_measurements.sensor_id;
+    (1 row)
+    ```
+
+1.  Connect to your Timescale database:
+
+    ```bash
+    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
+    ```
+
+1.  Recreate each continuous aggregate definition:
+
+    ```sql
+    CREATE MATERIALIZED VIEW <VIEW_NAME>
+    WITH (timescaledb.continuous) AS
+    <VIEW_DEFINITION>
+    ```
```

Comment:
What happens with the materialized data? I would assume that it is migrated, but then the name of the materialization table on the target might be different from the name on the source.

Reply:
I am guessing, it will get created AFRESH when the Cagg is computed on the target.